src/GRA.Web/Areas/MissionControl/Views/PerformerScheduling/Program.cshtml

```diff
@@ -1,5 +1,7 @@
 ﻿@model GRA.Controllers.ViewModel.MissionControl.PerformerScheduling.ProgramViewModel
 
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
Localization isn't needed on the MissionControl side of things.

Reply:
Fixed in commit 97b7d4757c5daf38b97e01a4b0f363bb290ee169